scripts/cluster

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Reset test-dirs
+rm -rf test-chain-*
+rm -f genesis.json
+
+go build -o polygon-edge .
+
+nodeid=$(./polygon-edge secrets init --data-dir test-chain- --num 4 | grep Node | head -n 1 | awk -F ' ' '{print $4}')
+
+./polygon-edge genesis --consensus ibft --ibft-validators-prefix-path test-chain- \
+  --bootnode /ip4/127.0.0.1/tcp/10001/p2p/"$nodeid" \
+  --premine 0x85da99c8a7c2c95964c8efd687e95e632fc533d6:1000000000000000000000 \
+  --block-gas-limit 10000000
+
+./polygon-edge server --data-dir ./test-chain-1 --chain genesis.json --grpc-address :10000 --libp2p :10001 --jsonrpc :10002 --seal --log-level DEBUG &
+./polygon-edge server --data-dir ./test-chain-2 --chain genesis.json --grpc-address :20000 --libp2p :20001 --jsonrpc :20002 --seal &
+./polygon-edge server --data-dir ./test-chain-3 --chain genesis.json --grpc-address :30000 --libp2p :30001 --jsonrpc :30002 --seal &
+./polygon-edge server --data-dir ./test-chain-4 --chain genesis.json --grpc-address :40000 --libp2p :40001 --jsonrpc :40002 --seal &
+
+# Uncomment to run with polybft
```

Comment:
Could we enhance script so that it contains flag which consensus we want to run, instead of (un)commenting code?